### PR TITLE
Update links point to the repo ocm.

### PR DESCRIPTION
**Pull request by xuezhaojun**
The related issue: https://github.com/open-cluster-management-io/OCM/issues/128

The fmt tool automatically delete some spaces in markdown files, but I also explicitly point out the changes using commits.

### CODE REVIEW

File: content/en/concepts/architecture.md [hunk 1]
```diff
@@ -87,14 +87,14 @@ managed clusters by leveraging on the Kubernetes' subproject [konnectivity](http
 Not only OCM will bring you a fluent user-experience of managing a number of
 clusters on ease, but also it will be equally friendly to further customization
 or second-time development. Every functionality working in OCM is expected to
-be freely-pluggable by modularizing the atomic capability into separated 
-building blocks, except for the mandatory core module named [registration](https://github.com/open-cluster-management-io/registration)
-which is responsible for controlling the lifecycle of a managed controller 
+be freely-pluggable by modularizing the atomic capability into separated
+building blocks, except for the mandatory core module named [registration](https://github.com/open-cluster-management-io/ocm/tree/main/cmd/registration)
```
Comment (xuezhaojun) [inline content/en/concepts/architecture.md:91]: Change to pointing to ocm repo the /cmd/registration

File: content/en/concepts/managedcluster.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ gracefully with permission and validity under control.
 
 
 Note that the functionality mentioned above are all managed by OCM's
-[registration](https://github.com/open-cluster-management-io/registration)
+[registration](https://github.com/open-cluster-management-io/ocm/tree/main/cmd/registration)
```
Comment (xuezhaojun) [inline content/en/concepts/managedcluster.md:91]: One of the changes.

File: content/en/concepts/managedclusterset.md [hunk 3]
```diff
@@ -242,23 +242,23 @@ rules:
...
+For easier management, we introduce a ManagedClusterSet called `default`.
+A `default` ManagedClusterSet will be automatically created initially. Any clusters not specifying a ManagedClusterSet will be added into the `default`.
 The user can move the cluster from the default clusterset to another clusterset using the command:
 ```
 clusteradm clusterset set target-clusterset --clusters cluster-name
 ```
 
 `default` clusterset is an alpha feature that can be disabled by disabling the feature gate in registration controller as:
-[`- "--feature-gates=DefaultClusterSet=false"`](https://github.com/open-cluster-management-io/registration-operator/commit/55bc274d795ad0befc71f05aecd08810a4abfba1#diff-1026afceb1a224783dbf517bc281e71c1640636f5f001338f8185a0b4398b3d9R51) 
+[`- "--feature-gates=DefaultClusterSet=false"`](https://github.com/open-cluster-management-io/ocm/commit/55bc274d795ad0befc71f05aecd08810a4abfba1#diff-1026afceb1a224783dbf517bc281e71c1640636f5f001338f8185a0b4398b3d9R51)
```
Comment (xuezhaojun) [inline content/en/concepts/managedclusterset.md:253]: One of the changes.

File: content/en/concepts/manifestwork.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ weight: 2
 
 ## What is `ManifestWork`
 
-`ManifestWork` is used to define a group of Kubernetes resources on the hub to be applied to the managed cluster. In the open-cluster-management project, a `ManifestWork` resource must be created in the cluster namespace. A work agent implemented in [work](https://github.com/open-cluster-management-io/work) project is run on the managed cluster and monitors the `ManifestWork` resource in the cluster namespace on the hub cluster.
+`ManifestWork` is used to define a group of Kubernetes resources on the hub to be applied to the managed cluster. In the open-cluster-management project, a `ManifestWork` resource must be created in the cluster namespace. A work agent implemented in [work](https://github.com/open-cluster-management-io/ocm/tree/main/cmd/work) project is run on the managed cluster and monitors the `ManifestWork` resource in the cluster namespace on the hub cluster.
```
Comment (xuezhaojun) [inline content/en/concepts/manifestwork.md:14]: One of the changes.

File: content/en/contribute/_index.md [hunk 5]
```diff
@@ -13,7 +13,7 @@ All contributions are welcome! Open Cluster Management uses the [Apache 2 licens
 
 - Join the [Mailing group](https://groups.google.com/g/open-cluster-management)
 - Attend an upcoming [Community meetings](https://github.com/open-cluster-management-io/community/projects/1)
-- Fork the [repository](https://github.com/open-cluster-management-io) you wish to work on. For example, [API](https://github.com/open-cluster-management-io/api),[Registration](https://github.com/open-cluster-management-io/registration), etc.
+- Fork the [repository](https://github.com/open-cluster-management-io) you wish to work on. For example, [API](https://github.com/open-cluster-management-io/api),[ocm](https://github.com/open-cluster-management-io/ocm), etc.
```
Comment (xuezhaojun) [inline content/en/contribute/_index.md:16]: One of the changes.

File: content/en/getting-started/installation/start-the-control-plane.md [hunk 6]
```diff
@@ -56,7 +56,7 @@ Call `clusteradm init`:
 ```
 
 The `clusteradm init` command installs the
-[registration-operator](https://github.com/open-cluster-management-io/registration-operator)
+[registration-operator](https://github.com/open-cluster-management-io/ocm/tree/main/cmd/registration-operator)
```
Comment (xuezhaojun) [inline content/en/getting-started/installation/start-the-control-plane.md:59]: One of the changes.

File: content/zh/blog/registration-deepdive/_index.md [hunk 7]
```diff
@@ -69,7 +69,7 @@ rules:
 
 ## 实现细节
 
-所有认证授权和秘钥管理的代码实现都在[registration](https://github.com/open-cluster-management-io/registration)组件中。大概的流程
+所有认证授权和秘钥管理的代码实现都在[registration](https://github.com/open-cluster-management-io/ocm/tree/main/cmd/registration)组件中。大概的流程
```
Comment (xuezhaojun) [inline content/zh/blog/registration-deepdive/_index.md:72]: One of the changes.

File: content/zh/concepts/architecture.md [hunk 8]
```diff
@@ -47,7 +47,7 @@ hub cluster的负担将会大大减轻，因为hub cluster既不需要处理来
 ### 模块化和可扩展性
 
 OCM不仅会给您带来流畅的用户体验，让您轻松管理多个集群，而且对进一步定制或二次开发同样友好。
-OCM中每一个功能，都可以通过将原子能力模块化到单独的构建块中，来实现自由拔插，除了了一个名为[registration](https://github.com/open-cluster-management-io/registration)的强制性核心模块，此模块负责控制managed控制器的生命周期，并导出基本的`ManagedCluster`模型。
+OCM中每一个功能，都可以通过将原子能力模块化到单独的构建块中，来实现自由拔插，除了了一个名为[registration](https://github.com/open-cluster-management-io/ocm/tree/main/cmd/registration)的强制性核心模块，此模块负责控制managed控制器的生命周期，并导出基本的`ManagedCluster`模型。
```
Comment (xuezhaojun) [inline content/zh/concepts/architecture.md:50]: One of the changes.

File: content/zh/concepts/managedcluster.md [hunk 9]
```diff
@@ -88,7 +88,7 @@ gracefully with permission and validity under control.
 
 
 Note that the functionality mentioned above are all managed by OCM's
-[registration](https://github.com/open-cluster-management-io/registration)
+[registration](https://github.com/open-cluster-management-io/ocm/tree/main/cmd/registration)
```
Comment (xuezhaojun) [inline content/zh/concepts/managedcluster.md:91]: One of the changes.

File: content/zh/concepts/managedclusterset.md [hunk 10]
```diff
@@ -242,23 +242,23 @@ rules:
...
+For easier management, we introduce a ManagedClusterSet called `default`.
+A `default` ManagedClusterSet will be automatically created initially. Any clusters not specifying a ManagedClusterSet will be added into the `default`.
 The user can move the cluster from the default clusterset to another clusterset using the command:
 ```
 clusteradm clusterset set target-clusterset --clusters cluster-name
 ```
 
 `default` clusterset is an alpha feature that can be disabled by disabling the feature gate in registration controller as:
-[`- "--feature-gates=DefaultClusterSet=false"`](https://github.com/open-cluster-management-io/registration-operator/commit/55bc274d795ad0befc71f05aecd08810a4abfba1#diff-1026afceb1a224783dbf517bc281e71c1640636f5f001338f8185a0b4398b3d9R51) 
+[`- "--feature-gates=DefaultClusterSet=false"`](https://github.com/open-cluster-management-io/ocm/commit/55bc274d795ad0befc71f05aecd08810a4abfba1#diff-1026afceb1a224783dbf517bc281e71c1640636f5f001338f8185a0b4398b3d9R51)
```
Comment (xuezhaojun) [inline content/zh/concepts/managedclusterset.md:253]: One of the changes.

File: content/zh/concepts/manifestwork.md [hunk 11]
```diff
@@ -11,7 +11,7 @@ weight: 2
 
 ## What is `ManifestWork`
 
-`ManifestWork` is used to define a group of Kubernetes resources on the hub to be applied to the managed cluster. In the open-cluster-management project, a `ManifestWork` resource must be created in the cluster namespace. A work agent implemented in [work](https://github.com/open-cluster-management-io/work) project is run on the managed cluster and monitors the `ManifestWork` resource in the cluster namespace on the hub cluster.
+`ManifestWork` is used to define a group of Kubernetes resources on the hub to be applied to the managed cluster. In the open-cluster-management project, a `ManifestWork` resource must be created in the cluster namespace. A work agent implemented in [work](https://github.com/open-cluster-management-io/ocm/tree/main/cmd/work) project is run on the managed cluster and monitors the `ManifestWork` resource in the cluster namespace on the hub cluster.
```
Comment (xuezhaojun) [inline content/zh/concepts/manifestwork.md:14]: One of the changes.

File: content/zh/contribute/_index.md [hunk 12]
```diff
@@ -13,7 +13,7 @@ All contributions are welcome! Open Cluster Management uses the [Apache 2 licens
 
 - Join the [Mailing group](https://groups.google.com/g/open-cluster-management)
 - Attend an upcoming [Community meetings](https://github.com/open-cluster-management-io/community/projects/1)
-- Fork the [repository](https://github.com/open-cluster-management-io) you wish to work on. For example, [API](https://github.com/open-cluster-management-io/api),[Registration](https://github.com/open-cluster-management-io/registration), etc.
+- Fork the [repository](https://github.com/open-cluster-management-io) you wish to work on. For example, [API](https://github.com/open-cluster-management-io/api),[ocm](https://github.com/open-cluster-management-io/ocm), etc.
```
Comment (xuezhaojun) [inline content/zh/contribute/_index.md:16]: One of the changes.

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**